test/types/string.js

```diff
@@ -561,7 +551,8 @@ describe('string', () => {
             Helper.validate(schema, [
                 ['joe@example.com', true],
                 ['joe@www.example.com', true],
-                ['joe@localhost', false, null, '"value" must be a valid email'],
```

Comment:
Assuming that test used to specify that `joe@localhost` was not a valid email, yes, this is an expect change. I'd actually like to see the same test, but `true`.